content/how-to/account-management/change-account-information/index.md

```diff
@@ -15,30 +15,41 @@ slug: change-account-information
 This article describes the various administrative account changes that you can
 request and the appropriate steps needed to complete them.
 
-### Dedicated customers
+### Dedicated Customers
 
-Dedicated customers can download the **Primary Contact Change** form in the
-[MyRackspace Control Panel](https://login.rackspace.com). For instructions
-about how to find the form, see the **Dedicated customers** section of [Change
-to a new primary contact](/support/how-to/change-to-new-primary-contact).
+You can download the **Primary Contact Change** form by using the following steps: 
 
-If you are the current primary contact for your account, you can update the
-primary contact information by selecting and editing the **Primary Contact
-Information** in your **Account Settings**.
+**Step 1.** Log in to the Rackspace Technology Portal <https://login.rackspace.com/>.
 
-Request the **Service Transfer** form from your account manager.
+**Step 2.** In the top navigation panel, select **Account >> Docs and Forms**.
+
+<img width="359" alt="Acc Docs   forms" src="Acc Docs & forms.png">
```

Comment:
image names should be lower case and no spaces

Reply:
updated